test/start-restart.js

```diff
@@ -70,8 +70,8 @@ if(cservice.isWorker){
         setTimeout(function() {
           assert.equal(
             cservice.workers.length,
-            2,
-            "2 workers expected, but " + cservice.workers.length + " found"
+            1,
+            "1 worker expected, but " + cservice.workers.length + " found"
```

Comment:
In my testing, this should actually be 0, but it isn't clear why this ever worked in 0.10... Perhaps this test should be disabled for now.